programming/typescript/challenges/index.js

```
@@ -0,0 +1,22 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+// Or import the entire module
+var Challenges = require("@lehungio/typescript");
+var helloAll = Challenges.greet('All'); // Output: "Hello, World!"
+console.log('Greeting result:', helloAll);
+/**
+* Challenge 1: Hello, World!
+* Demonstrates different ways to use the greet functionality
+*/
+var typescript_1 = require("@lehungio/typescript");
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)